src/v/cloud_storage/tests/read_replica_e2e_fixture.h

```diff
@@ -50,7 +50,7 @@ class read_replica_e2e_fixture
           std::vector<config::seed_server>{},
           test_directory(),
           app.sched_groups,
-          true,
+          false,
```

Comment:
not 100% clear that this is the right thing

Reply:
I also just did this for another test. I don't really understand why the directory is being destroyed twice

Reply:
Yeah it's weird. In this case it only seems to happen for the fixture instance that shares a lifetime w/ the test case. I don't really get it, but it also doesn't really seem worth blocking on.